lms/djangoapps/lms_xblock/mixin.py

```diff
@@ -26,6 +39,26 @@ def to_json(self, access_dict):
             return {unicode(k): access_dict[k] for k in access_dict}
 
 
+class EnumString(String):
```

Comment:
This could integrate more tightly with the LearnerVisibilityEnum, but subclassing String got things working more quickly.